### PR TITLE
added selection sort

### PR DESCRIPTION
**Pull request by Sloth-Panda**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes
(Write your answer here.)

### Description

(Write your answer here.)

This is selection sort implemented in js and explained with description too.
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
#2075

### CODE REVIEW

**Comment by Sloth-Panda** [comment]
@HarshCasper please take a look into it.

File: JavaScript/sort/SelectionSort.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+    return inputArray;
+}
+
+
+let inputArray = [12, 2, 34, 1, 19];
+selectionSort(inputArray);
+console.log(inputArray);
+
+(5)[1, 2, 12, 19, 34] /*output*/
```
Comment (HarshCasper) [inline JavaScript/sort/SelectionSort.js:33]: This is an entirely bad code example 🤦 

Keep the sample I/O in a multi-line comment just for better understanding. And add a line at the end of the file

File: JavaScript/sort/SelectionSort.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            let temp = inputArr[i];
+            inputArray[i] = inputArray[min];
+            inputArray[min] = temp;
+        }
+    }
+    return inputArray;
+}
+
+
+let inputArray = [12, 2, 34, 1, 19];
```
Comment (HarshCasper) [inline JavaScript/sort/SelectionSort.js:29]: Make the input dynamic

File: JavaScript/sort/SelectionSort.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+/*Selection sort is a simple sorting algorithm. ... The smallest element is
+ selected from the unsorted array and swapped with the leftmost element, and
+that element becomes a part of the sorted array. This process continues moving
+unsorted array boundary by one element to the right. */
```
Comment (HarshCasper) [inline JavaScript/sort/SelectionSort.js:4]: Justify the content and use proper sentences

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Pull request closed without merging**